docs/docs/dependency-specification.md

```diff
@@ -139,6 +139,20 @@ with the corresponding `add` call:
 poetry add https://example.com/my-package-0.1.0.tar.gz
 ```
 
+## `source` dependencies
+To dependent on a package located in the private repositry `my-private-repo`,
+you can use `source` property:
+```toml
+[[tool.poetry.source]]
+name = "my-private-repo"
+url = "https://example.com/"
+secondary = true
+
+[tool.poetry.dependencies]
+my-cool-package = { version = "*", source = "my-private-repo" }
+```
```

Comment:
```suggestion
```toml
[[tool.poetry.source]]
name = "foo"
url = "https://foo.bar/simple/"
secondary = true

[tool.poetry.dependencies]
my-cool-package = { version = "*", source = "foo" }
```
```

Let's keep the example consistent.